README.md

```diff
@@ -15,10 +15,10 @@ Dapr SDK for .NET allows you to implement the Virtual Actor model, based on the
 This repo builds the following packages:
 
 - Dapr.Client
-- Dapr.Client.Grpc
 - Dapr.AspNetCore
 - Dapr.Actors
 - Dapr.Actors.AspNetCore
+- Dapr.Configuration.Extensions
```

Comment:
We would normally recommend the naming structure `<your project>.Extensions.Configuration` for things that implement the `Microsoft.Extensions` types. Example: https://www.nuget.org/packages/Serilog.Extensions.Logging/3.0.2-dev-10272

Reply:
Done in ef3ba68